src/server/game/Entities/Object/Object.cpp

```diff
@@ -1529,13 +1529,32 @@ bool Position::IsPositionValid() const
 float WorldObject::GetGridActivationRange() const
 {
     if (ToPlayer())
+    {
+        if (ToPlayer()->IsOnCinematic())
+        {
+            return DEFAULT_VISIBILITY_INSTANCE;
+        }
         return IsInWintergrasp() ? VISIBILITY_DIST_WINTERGRASP : GetMap()->GetVisibilityRange();
+    }
     else if (ToCreature())
+    {
         return ToCreature()->m_SightDistance;
+    }
     else if (GetTypeId() == TYPEID_GAMEOBJECT && ToGameObject()->IsTransport())
-        return GetMap()->GetVisibilityRange();
+    {
+        if (isActiveObject())
```

Comment:
You can add if isActiveObject() to the else if above else if (GetTypeId() == TYPEID_GAMEOBJECT && ToGameObject()->IsTransport() && isActiveObject())

This way you don't need two else return 0.0f

Reply:
indeed

Reply:
done